README.md

```diff
@@ -32,7 +32,7 @@ for deprecation.
 This repo currently provides pre-configured Helm values sets for different scenarios as configuration
 [profiles](https://istio.io/docs/setup/kubernetes/additional-setup/config-profiles/), which act as a starting point for
 an Istio install and can be customized by creating customization overlay files or passing parameters when
-calling Helm. Similarly, the operator API uses the same profiles (expressed internally through the new API), which can be selected
+calling Helm. Similarly, the operator API uses the same profiles (expressed internally through the new API, e.g. compiled in [base profiles files](https://github.com/istio/operator/tree/master/data/profiles)), which can be selected
```

Comment:
We probably won't be shipping with compiled in.

Reply:
ok I would remove this part. Actually I hope to add some pointers to a list of base profiles that we provides. Maybe that would be added to istio.io?

Reply:
It's already in istio.io: https://istio.io/docs/setup/kubernetes/additional-setup/config-profiles/
We just need to stay in sync with these. We also have istioctl profile list that dumps the profiles for any given version of the operator.

Reply:
yep but some of the contents are not operator specific, for example the profile file name is referring to values.yaml file name

Reply:
ok, you convinced me :-)

Reply:
but please remove the compiled in bit since that's somewhat up in the air.